docs/schema.md

```diff
@@ -628,14 +781,14 @@ properties:
       type: DEPLOYER_IMAGE
 ```
 
----
-
 ## clusterConstraints
 
+This section is common to both `v1` and `v2` schemas.
+
 Use `clusterConstraints` to specify the requirements for the Kubernetes
-cluster. The requirements determine whether your application can be run on
-the cluster. For example, you can specify that the cluster must be set up to
-run a minimum number of replicas.
+cluster. The requirements determine whether your app can be run on the
+cluster. For example, you can specify that the cluster must be set up to run
+a minimum number of replicas.
 
 For example, the following `schema.yaml` specifies that the cluster must have
```

Comment:
Should create a subsection namely `resources`, to be parallel with the other subsections `istio` and `gcp`.